docs/src/starknet/account.md

```diff
@@ -88,6 +88,7 @@ $ sncast \
     --private-key 0x2 \
     --type oz
 ```
+The `--name` can be omitted as this is optional. A default name will be generated for the account.
```

Comment:
I'd not add that here, just in the other file you added.

Reply:
removed